common/lib/xmodule/xmodule/course_module.py

```diff
@@ -437,152 +439,308 @@ class CourseFields(object):
-                        "long_description": _("Use your course outline to build your first Section and Subsection."),
+                        "short_description": _("Include Syllabus in Course Handouts"),
+                        "long_description": _("Include a separate static page or PDF for the Course syllabus."),
                         "is_checked": False,
-                        "action_url": "CourseOutline",
-                        "action_text": _("Edit Course Outline"),
+                        "action_url": "",
+                        "action_text": "",
+                        "action_external": False,
+                    },
+                    {
+                        "short_description": _("Explain Pre-requisites"),
+                        "long_description": _(
+                            "State the Pre-requisites for the course clearly on the course syllabus."
+                        ),
+                        "is_checked": False,
+                        "action_url": "",
+                        "action_text": "",
+                        "action_external": False,
+                    },
+                    {
+                        "short_description": _("Explain time commitments"),
+                        "long_description": _(
+                            "State the time commitments for the course clearly on the course syllabus."
+                        ),
+                        "is_checked": False,
+                        "action_url": "",
+                        "action_text": "",
+                        "action_external": False,
+                    },
+                    {
+                        "short_description": _("Explain course deadlines"),
+                        "long_description": _(
+                            "State the course deadlines and due dates (including time zones)"
+                            "clearly on the course syllabus."
+                        ),
+                        "is_checked": False,
+                        "action_url": "",
+                        "action_text": "",
+                        "action_external": False,
+                    },
+                    {
+                        "short_description": _("Explain grading policy"),
+                        "long_description": _(
+                            "State the course grading policy and certificate requirements clearly on the course syllabus."
+                        ),
+                        "is_checked": False,
+                        "action_url": "",
+                        "action_text": "",
+                        "action_external": False,
+                    },
+                    {
+                        "short_description": _("Explain learning objectives"),
+                        "long_description": _(
+                            "State the learning objectives, goals, and course outcome(s) clearly on the course syllabus."
```

Comment:
State the learning objectives, the course goals, and the outcomes that learners can expect.